### ISSUE TITLE
Python int too large to convert to SQLite INTEGER when *Importing*

### ISSUE THREAD

**Issue by ionescu007** [joxeankoret/diaphora#3]
Hi,

When trying to import all functions (same IDB), I now receive this message instead. Note that diffing now worked correctly, this is only when importing:

[Wed Apr 08 07:32:18 2015] import_all(): Python int too large to convert to SQLite INTEGER
Traceback (most recent call last):
  File "C:/Program Files (x86)/IDA 6.7/plugins/diaphora-master/diaphora.py", line 1495, in import_all
    self.do_import_all(items)
  File "C:/Program Files (x86)/IDA 6.7/plugins/diaphora-master/diaphora.py", line 1476, in do_import_all
    self.import_items(items)
  File "C:/Program Files (x86)/IDA 6.7/plugins/diaphora-master/diaphora.py", line 1456, in import_items
    self.do_import_one(ea1, ea2)
  File "C:/Program Files (x86)/IDA 6.7/plugins/diaphora-master/diaphora.py", line 1424, in do_import_one
    cur.execute(sql, (ea2,))
OverflowError: Python int too large to convert to SQLite INTEGER

**Comment by joxeankoret** [comment]
Should be fixed with commit 7a9a6aa6d1fddcd21eef2aac29be318a41c3a81a

**Comment by ionescu007** [comment]
I had the same problem now after import is complete, at line 1435:

    cur.execute("delete from functions where address = ?", (ea, ))

I changed it to:

    cur.execute("delete from functions where address = ?", (str(ea), ))

And the problem seems to have gone away, but not sure if this is the right fix.

**Comment by joxeankoret** [comment]
Not the right patch, it should be this:

```
diff --git a/diaphora.py b/diaphora.py
index bd7f343..f6ab449 100644
--- a/diaphora.py
+++ b/diaphora.py
@@ -1476,7 +1476,7 @@ class CBinDiff:
     try:
       show_wait_box("Updating primary database...")
       for ea in to_import:
-        ea = int(ea)
+        ea = str(ea)
         new_func = self.read_function(ea)
         self.delete_function(ea)
         self.save_function(new_func)
```

**Comment by joxeankoret** [comment]
Should be fixed (I hope) with 9eb60b7bf95f7a659b8f3d29a83e2d31d6bd2b74
